kuma/wiki/tests/test_templates.py

```diff
@@ -128,35 +128,42 @@ def test_document_fallback_with_translation(self):
         revision(document=d2, is_approved=False, save=True)
         url = reverse('wiki.document', args=[d2.slug], locale='fr')
         response = self.client.get(url)
+        assert response.status_code == 200
+        assert 'public' in response['Cache-Control']
+        assert 's-maxage' in response['Cache-Control']
         doc = pq(response.content)
-        eq_(d2.title, doc('main#content div.document-head h1').text())
+        assert doc('main#content div.document-head h1').text() == d2.title
 
         # Fallback message is shown.
-        eq_(1, len(doc('#doc-pending-fallback')))
-        ok_('$translate' in doc('#edit-button').attr('href'))
+        assert len(doc('#doc-pending-fallback')) == 1
+        assert '$translate' in doc('#edit-button').attr('href')
         # Removing this as it shows up in text(), and we don't want to depend
         # on its localization.
         doc('#doc-pending-fallback').remove()
         # Included content is English.
-        eq_(pq(r.document.html).text(), doc('article#wikiArticle').text())
+        assert pq(r.document.html).text() == doc('article#wikiArticle').text()
 
     def test_document_fallback_no_translation(self):
         """The document template falls back to English if no translation
         exists."""
         r = revision(save=True, content='Some text.', is_approved=True)
         url = reverse('wiki.document', args=[r.document.slug], locale='fr')
         response = self.client.get(url)
+        assert response.status_code == 200
```

Comment:
I am seeing this common thing is checked in many tests. Is it possible to have a generalized function for this?
Something like
```python
def check_cahce(response):
     assert response.status_code == 200
     assert 'public' in response['Cache-Control']
     assert 's-maxage' in response['Cache-Control']
```

Then call `check_cahce` from all the function where its needed.

Reply:
@safwanrahman Good question. I've often wondered what would be best when I'm writing tests, to make things DRY ("Don't Repeat Yourself"), as you've indicated, or more directly readable as I've done. For tests, I've tended towards trying to make them as directly readable as possible, so there's no need to look-up/understand additional functions.